lonboard/geopandas.py

```diff
@@ -0,0 +1,296 @@
+
+        Keyword Args:
+            column : Name of column on dataframe to visualize on map.
+            cmap : Name of matplotlib colormap to use.
+            scheme : Name of a classification scheme defined by mapclassify.Classifier.
+            k : Number of classes to generate. Defaults to 6.
+            categorical : Whether the data should be treated as categorical or
+                continuous.
+            elevation : Name of column on the dataframe used to extrude each geometry or
+                an array-like in the same order as observations. Defaults to None.
+            extruded : Whether to extrude geometries using the z-dimension.
+            elevation_scale : Constant scaler multiplied by elevation value.
+            alpha : Alpha (opacity) parameter in the range (0,1) passed to
+                mapclassify.util.get_color_array.
+            layer_kwargs : Additional keyword arguments passed to lonboard.viz layer
+                arguments (either polygon_kwargs, scatterplot_kwargs, or path_kwargs,
+                depending on input geometry type).
+            map_kwargs : Additional keyword arguments passed to lonboard.viz map_kwargs.
+            classification_kwds : Additional keyword arguments passed to
+                `mapclassify.classify`.
+            nan_color : Color used to shade NaN observations formatted as an RGBA list.
+                Defaults to [255, 255, 255, 255]. If no alpha channel is passed it is
+                assumed to be 255.
+            color : single or array of colors passed to Layer.get_fill_color
+                or a lonboard.basemap object, or a string to a maplibre style basemap.
+            vmin : Minimum value for color mapping.
+            vmax : Maximum value for color mapping.
+            wireframe : Whether to use wireframe styling in deckgl.
+            tiles : Either a known string {"CartoDB Positron",
+                "CartoDB Positron No Label", "CartoDB Darkmatter",
+                "CartoDB Darkmatter No Label", "CartoDB Voyager",
+                "CartoDB Voyager No Label"}
+            highlight : Whether to highlight each feature on mouseover (passed to
+                lonboard.Layer's auto_highlight). Defaults to False.
+            m: An existing Map object to plot onto.
+
+        Returns:
+        lonboard.Map
+            a lonboard map with geodataframe included as a Layer object.
+        """
+        return _dexplore(
+            self._obj,
+            column,
+            cmap,
+            scheme,
+            k,
+            categorical,
+            elevation,
+            extruded,
+            elevation_scale,
+            alpha,
+            layer_kwargs,
+            map_kwargs,
+            classification_kwds,
+            nan_color,
+            color,
+            vmin,
+            vmax,
+            wireframe,
+            tiles,
+            highlight,
+            m,
+        )
+
+
+def _dexplore(
+    gdf,
+    column=None,
```

Comment:
For the internal function, we should make all of these keyword-only arguments to ensure we're passing the correct names below.
```suggestion
    *,
    column=None,
```